lawlibrary/urls.py

```diff
@@ -4,5 +4,8 @@
 
 urlpatterns = [
     path("", views.HomePageView.as_view(), name="home_page"),
+    path("judgments/", views.JudgmentListView.as_view(), name="judgment_list"),
+    path("judgments/<str:code>/", views.CourtDetailView.as_view(), name="court_detail"),
+    path("judgments/<str:code>/<int:year>/", views.YearView.as_view(), name="year"),
```

Comment:
Not crucial, but I suggest naming the route and the view class to as "CourtYearView" or similar. Year is rather unspecific.

Reply:
done